kubernetes/gaffer/templates/api/service.yaml

```diff
@@ -25,7 +25,7 @@ spec:
     app.kubernetes.io/component: api
   type: ClusterIP
   ports:
-  - name: http
+  - name: https
```

Comment:
If I understand this, this doesn't just enable https but forces you to use it? Could there be an option?

Reply:
Yeah this forces HTTPS. I guess it could be an option using if statements? I'm not too sure but could try it out. Would there be a reason to prefer HTTP over HTTPS though?

Reply:
I think perhaps an option might be better as it was probably HTTP before to make simple installs easier.

Reply:
This doesn't force HTTPS. This just changes the port you need to access on the ClusterIP to be routed to the http port on the api pod. All the traffic is still HTTP.

Before:
http://gaffer-api.svc:80 -> http://gaffer-api.pod:8080
Now:
http://gaffer-api.svc:443 -> http://gaffer-api.pod:8080

Reply:
My understanding is that in order for HTTPS to be used port 443 has to be used instead of 80. This in conjunction with the annotation additions in the ingress.yaml means that HTTPS is used instead of HTTP.

Reply:
Ignore